### ISSUE TITLE
Error in Part1 (xtb_rrho calculations) of CENSO_2.0

### ISSUE THREAD

**Issue by BeWater23** [grimme-lab/CENSO#59]
Hi,

when setting the evaluate_rrho = True in the gerenal setting, my calculations crash in Part1:Screening. 
As an error message when screening.py is executed I receive:

.../screening.py", line 339, in write_results2
    gxtb[id(conf)] += conf.results[self._name]["xtb_rrho"][
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable.

Everything works fine, until the results of the xtb_rrho calculations are evaluated. I was trying to find a solution but so far nothing worked... Do you have any idea what the issue could be?

Thanks a lot for your help!!

**Comment by lmseidler** [comment]
Hi, please provide me with some more information:
- How did you run censo (command line or from a python script)?
- Did you check if xtb ran correctly (you can check this by looking at the output files that should be generated by censo)?

Leopold

**Comment by BeWater23** [comment]
Sure:

- I ran it via the command line  (pathtoceno/censo2/CENSO/bin/censo -i crest_conformers.xyz --maxcores 48  --inprc pathtorc/.censo2rc) and a job submission script, since I run it on a cluster.
- Yes the xtb calculations worked perfectly fine I guess. For every conformer there are the directories with the xtb outputs. The censo output stops exactly here: 
Running xtb_rrho calculation for CONF2.
Running xtb_rrho calculation for CONF1.
Running xtb_rrho calculation for CONF3.
Running xtb_rrho calculation for CONF4.
Updated fuzzy threshold: 5.19 kcal/mol.

-------------------------------------- SCREENING RRHO RESULTS --------------------------------------
